detection-rules/attachment_archive_with_exe.yml

```diff
@@ -14,7 +14,7 @@ type: "rule"
 severity: "high"
 source: |
   type.inbound
-  and any(attachments, .file_extension in~ $file_extensions_common_archives
```

Comment:
File types and file extensions aren't 1:1, so you'll want to manually compare all the known `.file_type` values to `$file_extensions_common_archives` to validate overlap. Probably want an `or` here to be safe too